config/routes.rb

```diff
@@ -9,7 +9,8 @@
   root to: 'index#index', as: :index
 
   post  'phones/verification', to: 'phones#verify'
-  get   'security',     to: 'security#enable'
+  get   'security',                 to: 'security#enable'
+  get   'security_authentication',  to: 'security#security_authentication'
```

Comment:
We can make the route security#otp for now
but this will certainly move in the login process